### DISCUSSION TITLE
Add second pool crashes API

### DISCUSSION THREAD

**Discussion by riksarchen** [coinfoundry/miningcore#991] [General]
I'm currently running a pool on vertcoin with pplns, I want to add a solo pool but as soon as I add a second pool in config /api/pools stops responding. I can still access stats for the first pool by /api/pools/id but I cannot get any stats for second pool. Databases are populated with data though and the pool is working, it's just the api that don't work. 

Any ideas?

**Comment by oliverw** [comment]
Post the log output when starting the pool please.

**Reply by riksarchen** [reply]
You can disregard this post. I made an error in config. I just deleted the payoutSchedule from config and that caused it to crash. 

When I added it with empty brackets like this it worked.

I changed to:
"paymentProcessing": {
                "enabled": true,
                "minimumPayment": 1,
                "payoutScheme": "SOLO",
                "payoutSchemeConfig": [
                  [
                    []
                  ]
                ],
            }
